### DISCUSSION TITLE
s6-overlay-suexec: fatal: can only run as pid 1

### DISCUSSION THREAD

**Discussion by bogordesaincom** [serversideup/docker-php#87] [Q&A]
i got error like this

```
s6-overlay-suexec: fatal: can only run as pid 1
```

**Comment by jaydrogers** [comment]
I'm sorry, but in order for anyone to help you, you need to share a lot more about your environment.

Here's a guide how to ask for help: https://serversideup.net/how-to-ask-for-help/

**Comment by jaydrogers** [comment]
Closing this discussion in favor of a consolidated discussion on this topic: https://github.com/serversideup/docker-php/discussions/191
